docs/resources/project.md

```diff
@@ -71,65 +74,70 @@ resource "project" "myproject" {
 
 ### Required
 
-- **admin_privileges** (Block Set, Min: 1) (see [below for nested schema](#nestedblock--admin_privileges))
-- **display_name** (String) Also known as project name on the UI
-- **key** (String, ForceNew) The Project Key is added as a prefix to resources created within a Project. This field is mandatory and supports only 3 - 6 lowercase alphanumeric characters. Must begin with a letter. For example: us1a.
+- `admin_privileges` (Block Set, Min: 1) (see [below for nested schema](#nestedblock--admin_privileges))
+- `display_name` (String) Also known as project name on the UI
+- `key` (String) The Project Key is added as a prefix to resources created within a Project. This field is mandatory and supports only 3 - 10 lowercase alphanumeric and hyphen characters. Must begin with a letter. For example: us1a.
 
 ### Optional
 
-- **block_deployments_on_limit** (Boolean) Block artifacts deployment if storage quota is exceeded.
-- **description** (String)
-- **email_notification** (Boolean) Alerts will be sent when reaching 75% and 95% of the storage quota. Serves as a notification only and is not a blocker
-- **group** (Block Set) Project group. Element has one to one mapping with the [JFrog Project Groups API](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-UpdateGroupinProject) (see [below for nested schema](#nestedblock--group))
-- **id** (String) The ID of this resource.
-- **max_storage_in_gibibytes** (Number) Storage quota in GiB. Must be 1 or larger. Set to -1 for unlimited storage. This is translated to binary bytes for Artifactory API. So for 1TB quota, this should be set to 1024 (vs 1000) which will translate to 1099511627776 bytes for the API.
-- **member** (Block Set) Member of the project. Element has one to one mapping with the [JFrog Project Users API](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-UpdateUserinProject). (see [below for nested schema](#nestedblock--member))
-- **repos** (Set of String) List of existing repo keys to be assigned to the project. By default, `repos` is capped at 100 keys. (see [below for limitations on the maximum number of repos](#repoNumLimitations)). You can remove the system's cap by setting an environment value `REPO_LIMIT_OVERRIDE` to `true`. This setting will remove the restriction of maximum allowable elements in the `repos` attribute. The default value of `REPO_LIMIT_OVERRIDE` is `false`, e.g. To override use export REPO_LIMIT_OVERRIDE=true` in the shell.
-- **role** (Block Set) Project role. Element has one to one mapping with the [JFrog Project Roles API](https://www.jfrog.com/confluence/display/JFROG/Artifactory+REST+API#ArtifactoryRESTAPI-AddaNewRole) (see [below for nested schema](#nestedblock--role))
+- `block_deployments_on_limit` (Boolean) Block artifacts deployment if storage quota is exceeded.
```

Comment:
Is this plural artifacts or possessive artifact's deployment?

If its possessive it should be `Block the artifact's deployment if the storage quota is exceeded` 

It can also be a stylistic choice to avoid `the` so its up to you.